ui/pages/onboarding-flow/secure-your-wallet/skip-srp-backup-popover.js

```diff
@@ -105,12 +99,12 @@ export default function SkipSRPBackup({ handleClose }) {
               checked={checked}
               dataTestId="skip-srp-backup-popover-checkbox"
             />
-            <Typography
+            <Text
               className="skip-srp-backup-popover__details"
-              variant={TypographyVariant.H7}
+              variant={TextVariant.bodyXs}
```

Comment:
I think we probably want to use norm body here which is the default for Text so we can remove
```suggestion
```

Reply:
Removed `variant={TextVariant.bodyXs}`